stripe/src/test/java/com/stripe/android/view/ShippingInfoWidgetTest.kt

```diff
@@ -258,6 +258,20 @@ class ShippingInfoWidgetTest {
         assertEquals(countryAutoCompleteTextView.selectedCountry?.code, "US")
     }
 
+    @Test
+    fun getSelectedShippingCountry_whenShippingInfoProvided_returnsExpected() {
+        shippingInfoWidget.setAllowedCountryCodes(setOf("US", "CA"))
+        shippingInfoWidget.populateShippingInfo(SHIPPING_INFO_CA)
+        assertEquals(stateEditText.text.toString(), "Ontario")
+        assertEquals(cityEditText.text.toString(), "Ontario")
+        assertEquals(addressLine1EditText.text.toString(), "185 Berry St")
+        assertEquals(addressLine2EditText.text.toString(), "10th Floor")
+        assertEquals(phoneEditText.text.toString(), "416-759-0260")
+        assertEquals(postalEditText.text.toString(), "M4B1B5")
+        assertEquals(nameEditText.text.toString(), "Fake Name")
+        assertEquals(countryAutoCompleteTextView.selectedCountry?.code, "CA")
```

Comment:
The argument order for `assertEquals()` should be `assertEquals(expected, actual)`. Looks like this is `assertEquals(actual, expected)`. Can you swap the order for these lines?

Reply:
Sure thing. 👍